client/chat/action-creators.ts

```diff
@@ -279,25 +280,23 @@ export function getChatUserProfile(
 ): ThunkAction {
   return abortableThunk(spec, async (dispatch, getStore) => {
     const channelTargetId: `${SbChannelId}|${SbUserId}` = `${channelId}|${targetId}`
```

Comment:
nit: this is only used in one place now so you can just inline it and avoid needing to declare the type explicitly